documentation/tutorials/Demo_1_Basic_Run/run_pecan.qmd

```diff
@@ -0,0 +1,328 @@
+
+## 13.1 Plot Carbon Fluxes
+
+```{r plot-carbon-fluxes}
+# Plot Gross Primary Productivity (GPP) and Net Primary Productivity (NPP)
+plot(model_output$posix, model_output$GPP,
+  type = "l",
+  col = "green",
+  xlab = "Date",
+  ylab = "Carbon Flux (kg C m-2 s-1)",
+  main = "Carbon Fluxes Over Time"
+)
+lines(model_output$posix, model_output$NPP, col = "blue")
+legend("topright", legend = c("GPP", "NPP"), col = c("green", "blue"), lty = 1)
+```
+
+## 13.2 Plot Carbon Pools
+
+```{r plot-carbon-pools}
+# Plot Total Live Biomass and Total Soil Carbon
+plot(model_output$posix, model_output$TotLivBiom,
+  type = "l",
+  col = "darkgreen",
+  xlab = "Date",
+  ylab = "Carbon Pool (kg C m-2)",
+  main = "Carbon Pools Over Time"
+)
+lines(model_output$posix, model_output$TotSoilCarb, col = "brown")
+legend("topright", legend = c("Total Live Biomass", "Total Soil Carbon"), col = c("darkgreen", "brown"), lty = 1)
+```
+
+## 13.3 Plot Water Variables
+
+```{r plot-water-variables}
+# Plot Soil Moisture and Snow Water Equivalent
+plot(model_output$posix, model_output$SoilMoist,
+  type = "l",
+  col = "blue",
+  xlab = "Date",
+  ylab = "Soil Moisture (kg m-2)",
+  main = "Soil Moisture Over Time"
+)
+lines(model_output$posix, model_output$SWE, col = "lightblue")
+legend("topright", legend = c("Soil Moisture", "Snow Water Equivalent"), col = c("blue", "lightblue"), lty = 1)
+```
+
+## 13.4 Plot LAI and Biomass
+
+```{r plot-lai-biomass}
+# Plot Leaf Area Index (LAI) and Above Ground Wood
+plot(model_output$posix, model_output$LAI,
+  type = "l",
+  col = "darkgreen",
+  xlab = "Date",
+  ylab = "LAI (m2 m-2)",
+  main = "Leaf Area Index Over Time"
+)
+lines(model_output$posix, model_output$AbvGrndWood, col = "brown")
+legend("topright", legend = c("LAI", "Above Ground Wood"), col = c("darkgreen", "brown"), lty = 1)
+```
+
+# 14. Conclusion
+
+This notebook demonstrated how to set up, run, and analyze a PEcAn ecosystem model workflow programmatically. You can now modify parameters, try different models, or extend the analysis as needed.
+
+Try editing the `pecan.xml` file. Give it a new name and update the **settings_path** variable at the beginning of this Demo to point to the new file. See how the changes affect the model output!
+
+# 15. Clean Up Workflow Output (Optional)
+
+If you want to remove all files and directories created by this workflow and start fresh, you can run the following code. This will delete the entire output directory specified in your settings. **Use with caution!**
+
+```{r cleanup}
+# WARNING: This will permanently delete all workflow output files!
+# Uncomment the line below to enable cleanup.
+# fs::dir_delete(settings$outdir)
+```
+
+# 16. Session Info
+
+This section prints your R session information for reproducibility.
+
+```{r session-info}
+sessionInfo()
+```
```

Comment:
I recommend moving this to the top, so that it's available for debugging even if the notebook stops with an error before it gets to the end, and also calling `PEcAn.all::pecan_version()` in the same block.